src/components/Pages/Feedback/Feedback.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import Button from '../../layout/Buttons/Button';
+import FormElement from '../../layout/Form/FormElement';
+import Title from '../../layout/Typography/Title/Title';
+
+function Feedback() {
+  const [email, setemail] = useState('');
+  const [message, setmessage] = useState('');
+
+  const handleSubmit = event => {
+    event.preventDefault();
+    const user = {
+      username: email,
+      message: message
+    };
+    console.log('SUBMIT BABY', user);
+    localStorage.setItem('feedback', JSON.stringify(user));
+    setemail('');
+    setmessage('');
+  };
+
+  return (
+    <div>
+      <Title>Feedback</Title>
+      <p>
+        Dear User, your feedback is important for us. Please feel free to let us know what you think
+        about this app.
+      </p>
+      <form className="form" id="form">
+        <FormElement>
+          <input
```

Comment:
You have Button component - good, but for input and textarea you didn't add component...